html/js/guiutils/Tools.js

```diff
@@ -54,13 +54,36 @@ GTE.UI = (function (parentModule) {
             case GTE.MODES.PLAYER_ASSIGNMENT:
                 buttonToSwitch = "button-player-" + this.activePlayer;
                 break;
+            case GTE.MODES.MERGE:
+                if (this.ableToSwitchToISetMode()) {
```

Comment:
In the future (maybe, create an issue?) it would be nice if, besides having this check, the actual ISet buttons were disabled when this condition was not met.